Assets/Plugins/CountlySDK/Countly.cs

```diff
@@ -183,15 +183,15 @@ public async void Init(CountlyConfiguration configuration)
             RequestRepository requestRepo = new RequestRepository(requestDao, Configuration);
             ViewEventRepository eventViewRepo = new ViewEventRepository(viewEventDao, viewSegmentDao, Configuration);
             NonViewEventRepository eventNonViewRepo = new NonViewEventRepository(nonViewEventDao, nonViewSegmentDao, Configuration);
-            EventNumberInSameSessionDao eventNrInSameSessionDao = new EventNumberInSameSessionDao(auto, EntityType.EventNumberInSameSessions.ToString(), Configuration);
+
+            Dao<EventNumberInSameSessionEntity> eventNrInSameSessionDao = new Dao<EventNumberInSameSessionEntity>(auto, EntityType.EventNumberInSameSessions.ToString(), Configuration);

```

Comment:
"EventNumberInSameSessions" should be removed as a type from the "EntityType" enum. Just use a string here.